profile/faq.md

```diff
@@ -0,0 +1,82 @@
+# Frequently Asked Questions
+
+: Last Updated: 8 June 2025
+
+# General Questions
+
+## How do I use this?
+
+Our MVP product is a WordPress plugin, the FAIR Plugin. You can install it just like you would any other plugin.
```

Comment:
Nitpick: MVP product is redundant. Also, for maximum comprehensibility, can we just say "minimum viable product (MVP)"? For accessibility, it's best to expand an abbreviation the first time it appears on a page.